modules/appeals_api/spec/requests/v2/legacy_appeals_controller_spec.rb

```diff
@@ -6,33 +6,39 @@
   describe '#index' do
     context 'when only ssn provided' do
       it 'GETs legacy appeals from Caseflow successfully' do
-        VCR.use_cassette('caseflow/legacy_appeals_get_by_ssn') do
-          get_legacy_appeals(ssn: '502628285', file_number: nil)
-          expect(response).to have_http_status(:ok)
-          json = JSON.parse(response.body)
-          expect(json['data']).not_to be nil
-          expect(json['data'][0]['attributes']).to include 'latestSocSsocDate'
+        with_settings(Settings.modules_appeals_api, legacy_appeals: true) do
```

Comment:
Possibly unnecessary if you set the flag to `true` in config/settings.yml

Reply:
see above comment